tests/acceptance/install/02-InstallExtensionCest.php

```diff
@@ -21,28 +21,27 @@ public function activateWebservices(\AcceptanceTester $I)
 		$I->wantToTest('Activate the default webservices available in redCORE');
 		$I->doAdministratorLogin();
 		$I->comment('I enable basic authentication');
-		$I->amOnPage('administrator/index.php?option=com_plugins');
-		$I->waitForText('Plugins', 30, ['css' => 'H1']);
-		$I->fillField(['id' => 'filter_search'], 'redcore - system plugin');
-		$I->click(['css' => "button[data-original-title='Search']"]);
-		$I->waitForElement(['link' => 'redCORE - System plugin'], 60);
-		$I->click(['link' => 'redCORE - System plugin']);
-		$I->waitForText('Plugins: redCORE - System plugin', 30, ['css' => 'h1']);
+		$I->amOnPage('administrator/index.php?option=com_redcore&view=config&layout=edit&component=com_redcore');
+		$I->waitForText('redCORE - component Config', 30, ['css' => 'h1']);
 		$I->click(['link' => 'Webservice options']);
+		$I->waitForElementVisible(['id' => 'REDCORE_WEBSERVICES_OPTIONS']);
+		$I->executeJS("javascript:document.getElementById(\"REDCORE_WEBSERVICES_OPTIONS\").scrollIntoView();");
 		$I->selectOptionInRadioField('Enable webservices', 'Yes');
+		$I->executeJS("javascript:window.scrollBy(0,200);");
```

Comment:
😅 

Actually I found a way to remove floating bars: https://github.com/redCOMPONENT-COM/redSHOPB2B/blob/develop/tests/acceptance/install/2-InstallExtensionCest.php#L41-L76 , it will save us from this kind of issues.

I implemented in redSHOPB2B already and it works. I'll bring it to redCORE.